cli/src/funTest/kotlin/OrtMainFunTest.kt

```diff
@@ -160,6 +166,23 @@ class OrtMainFunTest : StringSpec() {
                 )
             }
         }
+
+        "Commands load OrtResults with resolved scopes" {
+            val cmd = AdvisorCommand()
+            val resultFile = File("src/funTest/assets/analyzer-result-with-dependency-graph.yml")
+
+            val result = cmd.readOrtResult(resultFile)
+
+            result.analyzer?.result shouldNotBeNull {
+                dependencyGraphs.keys should beEmpty()
+            }
+
+            val project = result.getProject(Identifier("Maven:com.vdurmont:semver4j:3.1.0"))
+
+            project.shouldNotBeNull {
```

Comment:
Nit: Ideally, this should also use the infix notation.

Reply:
@oheger-bosch looks like you need to rebase anyway to get rid of CI failures, and if doing so, would you mind to address my nit despite the approval?